### DISCUSSION TITLE
麥當勞X獵人

### DISCUSSION THREAD

**Discussion by yuan3x2** [1101-GS4719-C/Python-Community-GS4719-C#2218] [General]
前陣子有許多網紅或youtuber因為公關品的宣傳而被炎上，我本人也是獵人粉，沒想到此事會引起軒然大波，因此蠻好奇其他獵人粉對於此事的看法，會不會認為不被尊重或是認為沒差呢？

題外話，大家都買到卡片了嗎

**Comment by 112121py** [comment]
遠端代購我買到兩包了www
我覺得這次會炎上的那麼嚴重的其中一個原因，是因為網紅雖然有一些些不太恰當的言論，事後沒有道歉反而是發文酸網友，這個公關是有點不太好啦~ 但當然還有很多其他的因素，是個蠻值得討論一下的話題

**Comment by poyen0806** [comment]
我覺得事件本身是因為言論不太合適，而不是因為公關品的宣傳，但我還是會繼續看下去

![image](https://github.com/user-attachments/assets/bd18ac04-7154-4ad9-8ffc-bc697c188b90)

**Comment by Boying12** [comment]
這不僅是對於獵人這部作品的輕視，更是對於每一位熱愛這部作品的粉絲的不尊重。作為粉絲，我們希望看到對作品的真誠和尊重，而不是把它當作獲利的工具。這樣的行為只會傷害到那些真正熱愛這部作品的人的感情。
